### DISCUSSION TITLE
How to diable it for a short period of time

### DISCUSSION THREAD

**Discussion by BlueDragon320** [Jas-SinghFSU/HyprPanel#196] [Q&A] [answered]
Is there anyway i could disable for some time, as i want to use normal waybar for sometime so i can monitor more stuff.

**Comment by Jas-SinghFSU** [comment, marked as answer]
You can either:
1. Remove the following line from your `hyprpanel.conf`
```
exec-once = ags
```
2. Keep it running for notifications and menus (via keybinds) but disable the bar by following this part of the wiki: https://hyprpanel.com/configuration/panel.html#hiding-the-bar-on-specific-monitors

**Reply by BlueDragon320** [reply]
![image](https://github.com/user-attachments/assets/2236596c-1280-4b08-8392-08906489bc86)
Couldn't find the hyprpanel.conf

**Reply by Jas-SinghFSU** [reply]
Sorry I meant `hyprland.conf`.
